data-access/Models/ArtefactModels.cs

```diff
@@ -7,10 +7,8 @@ public class ArtefactModel
         public string Mask { get; set; }
         public string TransformMatrix { get; set; }
         public short ZIndex { get; set; }
```

Comment:
We don't need Institution, CatalogNumber1, or CatalogNumber2 in the artefact model, the ImagedObjectId will suffice.

Reply:
Yes, you are correct. Can you please open an issue to Talya show she removes them from the frontend DTO?

Reply:
No problem, I just did it.  You can merge this PR now.